components/user-header.js

```diff
@@ -180,8 +184,35 @@ function HeaderNym ({ user, isMe }) {
     : <NymView user={user} isMe={isMe} setEditting={setEditting} />
 }
 
+function SocialLink ({ name, id }) {
+  if (name === 'Nostr') {
+    const npub = hexToBech32(id)
+    return (
+      <Link className={styles.social} target='_blank' href={`https://snort.social/${npub}`} rel='noreferrer'>
```

Comment:
For nostr notes, we use https://nostr.com/ already since it asks the user which client they want to use.

So I think we should also use it here.
```suggestion
      <Link className={styles.social} target='_blank' href={`https://nostr.com/${npub}`} rel='noreferrer'>
```